tmt/steps/prepare/feature/fips-enable.yaml

```diff
@@ -0,0 +1,97 @@
+# On RHEL/CentosStream 8 and 9 FIPS is enabled as follows:
+#
+# 1. Install crypto-policies-scripts, dracut-fips and grubby
+# 2. Execute fips-mode-setup --enable
+# 3. Reboot
+#
+# https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/security_hardening/switching-rhel-to-fips-mode_security-hardening
+#
+# On RHEL/CentosStream 10 FIPS is enabled as follows:
+#
+# 1. Install grubby
+# 2. Add fips=1 and boot= (if needed) to bootloader kernel arguments [*]
+# 3. Reboot
+#
+# [*] fips-mode-setup is not longer available on RHEL/CentosStream 10
+#
+# https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/10-beta/html/security_hardening/switching-rhel-to-fips-mode
+---
+- name: Enable FIPS mode on RHEL/CentosStream 8, 9 or 10
+  hosts: all
+
+  tasks:
+
+    - name: Install crypto-policies-scripts and dracut-fips
+      ansible.builtin.dnf:
+        name:
+          - crypto-policies-scripts
+          - dracut-fips
+        state: present
+      when: ansible_facts['distribution_major_version'] | int < 10
```

Comment:
As already established by EPEL playbooks, we prefer using prepared variables rather than `ansible_facts`, e.g. https://github.com/teemtee/tmt/blob/main/tmt/steps/prepare/feature/epel-enable.yaml#L9 - `ansible_distribution_major_version | int < 10` would be aligned with existing code base.

The same would apply to other occurrences, like `ansible_facts['architecture']`.

Reply:
Fixed in [4a5bf44](https://github.com/teemtee/tmt/pull/3344/commits/4a5bf443ef34cea7cab9185f85479a951a78a91c).

Reply:
Resolved, all occurrences updated.